### PR TITLE
all: improve some error strings

### PR DESCRIPTION
**Pull request by trillom8**
Fixed the error string format according to https://go.dev/wiki/CodeReviewComments#error-strings. PTAL

### CODE REVIEW

File: cmd/clef/main.go [hunk 1]
```diff
@@ -583,7 +583,7 @@ func accountImport(c *cli.Context) error {
 	}
 	if first != second {
 		//lint:ignore ST1005 This is a message for the user
-		return errors.New("Passwords do not match")
+		return errors.New("passwords do not match")
```
Comment (rjl493456442) [inline cmd/clef/main.go:586]: Could you please revert this change? I guess it's some message being printed for users. I would prefer to keep it as now.

Comment (trillom8) [reply cmd/clef/main.go:586]: Done!

Comment (trillom8) [reply cmd/clef/main.go]: Okay Done!

**Review by rjl493456442 — APPROVED** [review]
